websites/F/Folding@home/presence.ts

```diff
@@ -1,29 +1,33 @@
-var presence = new Presence({
-  clientId: "704385469856612523"
+const presence = new Presence({
+  clientId: "812025934617509949"
 });
+let points: string,
+  progress: string;
+presence.on("iFrameData", (data: IFrameData) => {
+  points = data.iframe_d.points,
+  progress = data.iframe_d.progress;
 
-let cpuUsage: any, contributingProject: any;
+});
 
 presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
     largeImageKey: "logo"
   };
-  if (document.location.hostname == "client.foldingathome.org") {
-    cpuUsage = document.querySelector(
-      "div.ui-progressbar-value.ui-widget-header.ui-corner-left"
-    );
-    contributingProject = document.querySelector("a.sbSelector");
-
-    presenceData.details = "Contributing to: " + contributingProject.innerText;
-    presenceData.state = "Project Progress: " + cpuUsage.innerText;
-  } else {
-    presenceData.details = "Can't read page";
-  }
```

Comment:
wait wait wait why did you removed it and changing to to less info??

Reply:
It contains almost all the same info just wait less lines of code (and without the can't read page).

Reply:
then do like `Project progress` etc